NZSLDict/Classes/DetailViewController.swift

```diff
@@ -34,53 +34,83 @@ class DetailViewController: UIViewController, UISplitViewControllerDelegate, UIN
-        videoView.backgroundColor = UIColor.black
-        view.insertSubview(videoView, belowSubview: diagramView)
+        let dvTop = diagramView.topAnchor.constraint(equalTo: navigationBar.bottomAnchor, constant: 16)
+        let dvLead = diagramView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 16)
+        let dvTrailing = diagramView.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -16)
+
+        videoView = UIView()
+        videoView.translatesAutoresizingMaskIntoConstraints = false
+        videoView.backgroundColor = .black
+        view.addSubview(videoView)
+
+        let vvTop = videoView.topAnchor.constraint(equalTo: diagramView.bottomAnchor, constant: 10)
+        let vvLead = videoView.leadingAnchor.constraint(equalTo: view.leadingAnchor)
+        let vvTrailing = videoView.trailingAnchor.constraint(equalTo: view.trailingAnchor)
+        let vvBottom = videoView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
+
+        let vvHeight = videoView.heightAnchor.constraint(equalTo: diagramView.heightAnchor, multiplier: 1)
+        let dvHeight = diagramView.heightAnchor.constraint(equalTo: videoView.heightAnchor, multiplier: 1)
+
+        NSLayoutConstraint.activate([
+            dvTop, dvLead, dvTrailing, vvTop, vvLead, vvTrailing, vvBottom, vvHeight, dvHeight
+        ])
 
         playButton = UIButton(type: .roundedRect)
-        playButton.frame = CGRect(x: 0, y: (videoView.bounds.size.height - 40) / 2, width: videoView.bounds.width, height: 40)
-        playButton.titleLabel?.textAlignment = .center
-        playButton.autoresizingMask = [.flexibleLeftMargin, .flexibleRightMargin, .flexibleTopMargin, .flexibleBottomMargin]
-        playButton.setTitle("Play Video", for: UIControl.State())
-        playButton.setTitle("Playing videos requires access to the Internet.", for: .disabled)
-        playButton.setTitleColor(UIColor.white, for: .disabled)
-
-        playButton.addTarget(self, action: #selector(DetailViewController.startPlayer(_:)), for: .touchUpInside)
+        playButton.translatesAutoresizingMaskIntoConstraints = false
         videoView.addSubview(playButton)
 
-        if #available(iOS 10.0, *) {
-            playerView.updatesNowPlayingInfoCenter = false
-        }
+        playButton.setTitle("Play Video", for: .normal)
+        playButton.setTitle("Playing videos requires access to the Internet", for: .disabled)
+
+        playButton.setTitleColor(.white, for: .disabled)
+        playButton.addTarget(self, action: #selector(startPlayer), for: .touchUpInside)
 
-        self.view = view
+        let pbCenterX = playButton.centerXAnchor.constraint(equalTo: videoView.centerXAnchor)
+        let pbCenterY = playButton.centerYAnchor.constraint(equalTo: videoView.centerYAnchor)
+
+        NSLayoutConstraint.activate([
+            pbCenterX, pbCenterY
+        ])
+
+
+//        view.insertSubview(videoView, belowSubview: diagramView)
```

Comment:
Can we remove the commented out line if it's no longer necessary?

Reply:
Ah, yes I forgot about that, I left that in as I was going to ask about it, I'm not quite sure why thats in there, I was just going to remove but I wanted to check if there was a reason for it.